cli/cmd/postgres.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright © 2022 CFC4N <cfc4n.cs@gmail.com>
+
+*/
+package cmd
+
+import (
+	"context"
+	"ecapture/user"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+var postgresConfig = user.NewPostgresConfig()
+
+//postgres Cmd represents the postgres command
+var postgresCmd = &cobra.Command{
+	Use:   "postgres",
+	Short: "capture sql queries from postgres 10+.",
+	Run:   postgresCommandFunc,
+}
+
+func init() {
+	postgresCmd.PersistentFlags().StringVarP(&postgresConfig.PostgresPath, "postgres", "m", "/usr/bin/postgres", "postgres binary file path, use to hook")
+	postgresCmd.PersistentFlags().Uint64VarP(&postgresConfig.Offset, "offset", "", 0, "0x710410")
```

Comment:
These values should not be constant. Why not use symbol name?

Reply:
will take a look and change it ~

Reply:
And I think I should drop `offset`